### PR TITLE
Cherrypick recent docs prs to docs branch

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

This PR cherry-picks recent docs PRs so the AWX documentation includes recent changes to remove references to controller etc.

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - Docs

##### AWX VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: awx_collection/tools/roles/template_galaxy/templates/README.md.j2 [hunk 1]
```diff
@@ -46,8 +46,7 @@ You can specify authentication by a combination of either:
  - host, OAuth2 token
 
 The OAuth2 token is the preferred method. You can obtain a token via the
-AWX CLI [login](https://docs.ansible.com/ansible-tower/latest/html/towercli/reference.html#awx-login)
-command.
+``login`` command with the AWX CLI.
```
Comment (tvo318) [inline awx_collection/tools/roles/template_galaxy/templates/README.md.j2:49]: Is the command not called "awx-login"?

**Review by oraNod — APPROVED** [review]
I think the ci failures are related to the branch naming. In any case I think it's safe to ignore them for this PR because the commits all passed ci on the devel branch from other PRs. Also we're merging to a temporary branch that will be used only to build docs on RTD - it won't go into a build or anything.

File: docs/docsite/rst/administration/index.rst [hunk 2]
```diff
@@ -13,10 +13,7 @@ Likewise content in this guide can be removed or replaced if it applies to funct
 
 **Join us online**
 
-We talk about AWX documentation on Matrix at `#docs:ansible.im <https://matrix.to/#/#docs:ansible.im>`_ and on libera IRC at ``#ansible-docs`` if you ever want to join us and chat about the docs!
-
-You can also find lots of AWX discussion and get answers to questions at `forum.ansible.com <https://forum.ansible.com/>`_.
-
+Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide<communication>` to learn how to join the conversation!
```
Comment (tvo318) [inline docs/docsite/rst/administration/index.rst:16]:
```suggestion
Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide <communication>` to learn how to join the conversation!
```

Comment (tvo318) [reply docs/docsite/rst/administration/index.rst:16]: I am pretty sure this is missing a space.

File: docs/docsite/rst/contributor/index.rst [hunk 3]
```diff
@@ -10,13 +10,14 @@ There are so many ways you can contribute to AWX.
 
 **Join us online**
 
-You can chat with us and ask questions on Matrix at `#awx:ansible.com <https://matrix.to/#/#awx:ansible.com>`_ or visit the `Ansible Community Forum <https://forum.ansible.com/c/project/7/>`_ to find contributor resources.
+Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide<communication>` to learn how to join the conversation!
```
Comment (tvo318) [inline docs/docsite/rst/contributor/index.rst:13]:
```suggestion
Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide <communication>` to learn how to join the conversation!
```

File: docs/docsite/rst/contributor/report_issues.rst [hunk 4]
```diff
@@ -9,14 +9,13 @@ as possible. Version information, and an accurate reproducing scenario are criti
 
 Be sure to attach the ``component:docs`` label to your issue. These labels are determined by the template data. Please use the template and fill it out as accurately as possible.
 
-Please don't use the issue tracker as a way to ask how to do something. Instead, discuss it on on the `Ansible Community Forum <https://forum.ansible.com/c/project/7/>`_, or you can chat with us and ask questions on Matrix at `#awx:ansible.com <https://matrix.to/#/#awx:ansible.com>`_.
+Please don't use the issue tracker as a way to ask how to do something. Instead, discuss it on on the Ansible Forum, or you can chat with us and ask questions on Matrix. See the :ref:`Communication guide<communication>` for details.
```
Comment (tvo318) [inline docs/docsite/rst/contributor/report_issues.rst:12]:
```suggestion
Please don't use the issue tracker as a way to ask how to do something. Instead, discuss it on on the Ansible Forum, or you can chat with us and ask questions on Matrix. See the :ref:`Communication guide <communication>` for details.
```

File: docs/docsite/rst/quickstart/index.rst [hunk 5]
```diff
@@ -12,9 +12,7 @@ Likewise content in this guide can be removed or replaced if it applies to funct
 
 **Join us online**
 
-We talk about AWX documentation on Matrix at `#docs:ansible.im <https://matrix.to/#/#docs:ansible.im>`_ and on libera IRC at ``#ansible-docs`` if you ever want to join us and chat about the docs!
-
-You can also find lots of AWX discussion and get answers to questions at `forum.ansible.com <https://forum.ansible.com/>`_.
+Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide<communication>` to learn how to join the conversation!
```
Comment (tvo318) [inline docs/docsite/rst/quickstart/index.rst:15]:
```suggestion
Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide <communication>` to learn how to join the conversation!
```

File: docs/docsite/rst/release_notes/index.rst [hunk 6]
```diff
@@ -12,9 +12,7 @@ Likewise content in this guide can be removed or replaced if it applies to funct
 
 **Join us online**
 
-We talk about AWX documentation on Matrix at `#docs:ansible.im <https://matrix.to/#/#docs:ansible.im>`_ if you ever want to join us and chat about the docs!
-
-You can also find lots of AWX discussion and get answers to questions on the `Ansible Community Forum <https://forum.ansible.com/c/project/7/>`_.
+Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide<communication>` to learn how to join the conversation!
```
Comment (tvo318) [inline docs/docsite/rst/release_notes/index.rst:15]:
```suggestion
Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide <communication>` to learn how to join the conversation!
```

File: docs/docsite/rst/upgrade_migration/index.rst [hunk 7]
```diff
@@ -12,9 +12,7 @@ Likewise content in this guide can be removed or replaced if it applies to funct
 
 **Join us online**
 
-We talk about AWX documentation on Matrix at `#docs:ansible.im <https://matrix.to/#/#docs:ansible.im>`_ and on libera IRC at ``#ansible-docs`` if you ever want to join us and chat about the docs!
-
-You can also find lots of AWX discussion and get answers to questions at `forum.ansible.com <https://forum.ansible.com/>`_.
+Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide<communication>` to learn how to join the conversation!
```
Comment (tvo318) [inline docs/docsite/rst/upgrade_migration/index.rst:15]:
```suggestion
Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide <communication>` to learn how to join the conversation!
```

File: docs/docsite/rst/userguide/index.rst [hunk 8]
```diff
@@ -13,9 +13,7 @@ Likewise content in this guide can be removed or replaced if it applies to funct
 
 **Join us online**
 
-We talk about AWX documentation on Matrix at `#docs:ansible.im <https://matrix.to/#/#docs:ansible.im>`_ and on libera IRC at ``#ansible-docs`` if you ever want to join us and chat about the docs!
-
-You can also find lots of AWX discussion and get answers to questions at `forum.ansible.com <https://forum.ansible.com/>`_.
+Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide<communication>` to learn how to join the conversation!
```
Comment (tvo318) [inline docs/docsite/rst/userguide/index.rst:16]:
```suggestion
Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide <communication>` to learn how to join the conversation!
```

**Review by tvo318 — APPROVED** [review]
I only found a few small things. Otherwise, it's good to go. Thanks for taking care of this cleanup work!

**Comment by oraNod** [comment]
@tvo318 Hey, sorry but the PR was already merged. I figured it was good to go since the changes have already been merged on to devel.

Could you create a new PR against devel that incorporates your suggestions here? I can take care of the cherry-pick to the temp docs branch. But I think it would be good to make sure devel and the temp docs branch stay in sync. Know what I mean?

File: docs/docsite/rst/contributor/index.rst (hunk 3 above)
Comment (samccann) [reply docs/docsite/rst/contributor/index.rst:13]: verified on the published docs that this seems to work with or without the spaces.

**Comment by samccann** [comment]
So the spaces aren't necessary above. That leaves only the change needed for the collection, which doesn't need to be cherry-picked here as it's not part of documentation afaik.
(I was probably overzealous with my cherry-picks in this PR).

**Comment by oraNod** [comment]
> So the spaces aren't necessary above. That leaves only the change needed for the collection, which doesn't need to be cherry-picked here as it's not part of documentation afaik. (I was probably overzealous with my cherry-picks in this PR).

Thanks for checking all that @samccann 

@gundalow I wonder if we should now lock the temp docs build branch in the repo settings? That would make it read only and prevent any further pushes, which might be closer to the "frozen" branch that we're going for.

It might also be worth letting the community know about the fact docs won't be updated while the re-architecture / modernization stuff is going on. What do folks think? Forum and bullhorn?

**Comment by samccann** [comment]
@oraNod - fwiw - I posted about the frozen docs already as a [comment](https://forum.ansible.com/t/streamlining-awx-releases/6894/22?u=samccann) to the AWX releases post.

**Comment by gundalow** [comment]
FYI, the `temp-docs-build` branch is now locked

**Pull request merged**